tomopy/prep/alignment.py

```diff
@@ -339,6 +335,7 @@ def align_joint(
             conv[n] = np.linalg.norm(err)
 
         if save:
+            import dxchange
             dxchange.write_tiff(prj, 'tmp/iters/prj/prj')

```

Comment:
Completely removing `dxchange` requires copying three functions from [`dxchange.writer`](https://dxchange.readthedocs.io/en/latest/_modules/dxchange/writer.html#write_tiff) because this function does stuff like automatically preventing existing files from being overwritten.